pkg/logproto/logproto.proto

```diff
@@ -188,7 +188,11 @@ message SeriesResponse {
 }
 
 message SeriesIdentifier {
-  map<string, string> labels = 1;
+  message LabelsEntry {
+    string key = 1;
+    string value = 2;
+  }
```

Comment:
Is there any advantage of making the message inline? I assume it should make it explicit that the type is only used in conjunction with SeriesIdentifier, right?

Reply:
> I assume it should make it explicit that the type is only used in conjunction with SeriesIdentifier

Yes. That's the whole reason.